machine_management/user_infra/adding-compute-user-infra-general.adoc

```diff
@@ -0,0 +1,32 @@
+[id="adding-compute-user-infra-general"]
+= Adding compute machines to clusters with user-provisioned infrastructure
+include::modules/common-attributes.adoc[]
+:context: adding-compute-user-infra-general
+
+toc::[]
+
+You can add compute machines to a cluster on user-provisioned infrastructure either as part of the installation process or after installation. The post-installation process requires some of the same configuration files and parameters that were used during installation.
+
+== Adding compute machines to Amazon Web Services
```

Comment:
All of these headings should have explicit anchors

Reply:
Thank you :upside_down_face: